optimum/intel/openvino/modeling_diffusion.py

```diff
@@ -447,9 +447,8 @@ def __init__(
 
     def _create_inference_request(self):
         if self.request is None:
-            logger.info("Compiling the encoder and creating the inference request ...")
```

Comment:
It would be great to rename the `_create_inference_request` method to `_compile` for stable diffusion models as well to keep uniformity

Reply:
Fixed